client/CPlayerInterface.cpp

```diff
@@ -662,7 +662,11 @@ void CPlayerInterface::battleStart(const CCreatureSet *army1, const CCreatureSet
 	if ((replayAllowed && useQuickCombat) || forceQuickCombat)
 	{
 		autofightingAI = CDynLibHandler::getNewBattleAI(settings["server"]["friendlyAI"].String());

```

Comment:
We select battle AI for autocombat depending on settings so stupid and empty AI should also work here so your changes should be backward compatible or you have to extend all these AIs

Reply:
emptyAI is adventure map AI unless I miss something?

Reply:
No, exactly there is a request to make some dummy adventure AI